src/zoid/buttons/component.jsx

```diff
@@ -262,6 +290,8 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
                 required: false,
                 default:  () => noop,
                 decorate: ({ props, value = noop }) => {
+                    logLatencyInstrumentationPhase({ buttonSessionID: props.buttonSessionID, phase: 'buttons-first-render' });
```

Comment:
I wonder if we should move this `'buttons-first-render'` start phase to the top of the prerenderTemplate instead of decorating `onInit`. Could you test locally and see which one happens first? 

Ex:

```js
prerenderTemplate: ({ state, props, doc, event }) => {

    // start
    logLatencyInstrumentationPhase({ buttonSessionID, phase: 'buttons-first-render' });

    event.on(EVENT.PRERENDERED, () => {
        // end
        logLatencyInstrumentationPhase({ buttonSessionID, phase: 'buttons-first-render-end' });
```

Reply:
@rubenfig and I synced up and learned that onInit() is called first so we will stick with that.